sample_app/Gemfile

```diff
@@ -7,7 +7,11 @@ gem 'aws-sdk-rails', path: '../'
 # gem 'aws-sessionstore-dynamodb', path: '../../aws-sessionstore-dynamodb-ruby'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails' #, path: '../../rails'
+gem 'rails', '~> 7.0'
```

Comment:
I prefer to keep the comment (for development)

Reply:
These are mostly changes from running a `rails app:upgrade` - we could instead create a new app (i think in particular rails 7 does not use webpack by default) and add our customizations in instead so we get a cleaner rails 7 app.  Thoughts?

Reply:
I didn't know about that command. I'm sure it's fine.